spec/requests/liquid_embeds_spec.rb

```diff
@@ -17,7 +17,7 @@
     it "renders 404 if improper tweet" do
       expect do
         get liquid_embed_path("tweet", args: "improper")
-      end.to raise_error(ActionView::Template::Error)
```

Comment:
This fixes the spec failure that CI caught from the first commit in this PR. I recognize that I'm saying this with the benefit of hindsight, but the fact that this was expecting a completely unrelated exception class should likely have been a hint that running all this in the template was a bit confusing — this isn't the exception class that the code under test raises.

Effectively, this expectation required that the work be done inside the template.